InterviewView.swift

```diff
@@ -0,0 +1,39 @@
+
+import UIKit
+
+class ViewController: UIViewController {
+    var output: ProfileViewOutput?
+    let resultButton = UILabel()
+    
+    required init?(coder: NSCoder) {
```

Comment:
No init method but output is not initialized, add init that getting argument to initialize output field